documentation/Using-the-SDK/Creating-a-table.md

```diff
@@ -9,8 +9,12 @@ including cookers that may not necessarily be shipped with said `Table`, to buil
 exposes data through cookers, then you can author a table to leverage said data.
 
 Creating a table involves two keys steps:
-* [Declaring the Table](#declaring-the-table)
-* [Integrating the Table with our Data-Processing Pipeline](#integrating-the-table-with-our-data-processing-pipeline)
+- [Creating a Table](#creating-a-table)
+  - [Declaring the Table](#declaring-the-table)
+  - [Integrating the Table with our Data-Processing Pipeline](#integrating-the-table-with-our-data-processing-pipeline)
+- [Examples](#examples)
```

Comment:
Do we want these links?

Reply:
I don't care, they were another auto add. I'll just remove them for now.